server/routes/api.js

```diff
@@ -27,6 +27,39 @@ router.get('/asset-compute-endpoint', async function(req, res) {
     });
 });
 
+
+router.get('/get-journal-url', async function(req, res) {
+    try {
+        if (!assetComputeDevTool) {
+            assetComputeDevTool = await setupAssetComputeDevTool();
+        }
+    }  catch(e) {
+        console.log(e);
+        return res.status(500).send({
+            message: 'Error setting up devtool - journal url'
+        });
+    }
+    const journalUrl = assetComputeDevTool && assetComputeDevTool.assetCompute 
+                            && assetComputeDevTool.assetCompute.journal;
+    res.json({ journalUrl });
+});
+
+router.post('/check-jrnl-ready', async function(req, res) {
```

Comment:
Do we really need to abreviate `journal` with `jrnl` here in the API endpoint? I mean `check-journal-ready` is not toooo long (btw it reminds more of an rpc-type url because of the verb :) "check").

Reply:
Also, I wonder why using a POST? You're not creating a journal (POST is meant for creating/adding things, usually), you're more checking/polling for its existence. As far as I see, your calls do not create the journal (or did I miss a line?). I would turn this into a GET... ?

Reply:
(This may also give you the opportunity to merge the two endpoints and poll it, instead of having 2 different endpoints. /journal could return a url only when the journal is ready).